cmd/frostfs-adm/internal/modules/root.go

```diff
@@ -34,7 +33,9 @@ func init() {
 	// use stdout as default output for cmd.Print()
 	rootCmd.SetOut(os.Stdout)
 
-	rootCmd.PersistentFlags().StringP(configFlag, "c", "", "Config file")
+	rootCmd.PersistentFlags().StringP(commonflags.ConfigFlag, commonflags.ConfigFlagShorthand, "", commonflags.ConfigFlagUsage)
+	rootCmd.PersistentFlags().BoolP(commonflags.Verbose, commonflags.VerboseShorthand, false, commonflags.VerboseUsage)
+	_ = viper.BindPFlag(commonflags.Verbose, rootCmd.PersistentFlags().Lookup(commonflags.Verbose))
```

Comment:
Can we factor out common flags in a separate commit? And add `netmap-candidates` in the next one.

Reply:
Done.